### PR TITLE
Implement fixed python version in .toml

### PR DESCRIPTION
**Pull request by Daxtek**
## Description

Address this [issue](https://github.com/swappiehq/laakso/pull/4729/files#r1315507995) to implement a fixed version inisde the .toml file for python update. 

## Checklist
- [ ] After merging this I have tested this by manually triggering the action in one of the internal projects

### CODE REVIEW

File: update-asdf-and-dockerfile/action.yml [hunk 1]
```diff
@@ -116,7 +116,7 @@ runs:
       shell: bash
       run: |
         test -f pyproject.toml && \
-        sed -i "s/^python =.*/python = \"^${{ env.LATEST_VERSION }}\"/" pyproject.toml && \
+        sed -i "s/^python =.*/python = \"${{ env.LATEST_VERSION }}\"/" pyproject.toml && \
```
Comment (artjom-vasin) [inline update-asdf-and-dockerfile/action.yml:119]: do we need `~` here?

**Review by artjom-vasin — APPROVED** [review]

File: update-asdf-and-dockerfile/action.yml (hunk 1 above)
Comment (Daxtek) [reply update-asdf-and-dockerfile/action.yml:119]: If we want a fixed version we would not need the `~` : https://python-poetry.org/docs/cli/#add

**Comment by Daxtek** [comment]
Will test rerunning the action

File: update-asdf-and-dockerfile/action.yml (hunk 1 above)
Comment (artjom-vasin) [reply update-asdf-and-dockerfile/action.yml:119]: Now I know it, thank you

**Pull request merged**